src/GeoContacts/GeoContacts/Helpers/CommonConstants.cs

```diff
@@ -15,10 +15,23 @@ public class CommonConstants
         #endregion
 
         #region AD
+        public const bool USE_MSFT = false;
```

Comment:
Can you set make a string to replace on the build server that this checks..

```
string string AppCenterUseMSFT = "AC_USEMSFT_NO";
public bool USE_MSFT = "AC_USEMSFT" == AppCenterUseMSFT;
```

and I will swap: AC_USEMSFT_NO with AC_USEMSFT